blog/electron-30-0.md

```diff
@@ -0,0 +1,96 @@
+---
+title: Electron 30.0.0
+date: 2024-04-16T00:00:00.000Z
+authors:
+  - name: clavin
+    url: 'https://github.com/clavin'
+    image_url: 'https://github.com/clavin.png?size=96'
+  - name: VerteDinde
+    url: 'https://github.com/vertedinde'
+    image_url: 'https://github.com/vertedinde.png?size=96'
+slug: electron-30-0
+---
+
+Electron 30.0.0 has been released! It includes upgrades to Chromium `124.0.6367.49`, V8 `12.4`, and Node.js `20.11.1`.
+
+---
+
+The Electron team is excited to announce the release of Electron 30.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://releases.electronjs.org/releases/stable). Continue reading for details about this release.
+
+If you have any feedback, please share it with us on [Twitter](https://twitter.com/electronjs) or [Mastodon](https://social.lfx.dev/@electronjs), or join our community [Discord](https://discord.com/invite/electronjs)! Bugs and feature requests can be reported in Electron's [issue tracker](https://github.com/electron/electron/issues).
+
+## Notable Changes
+
+### Highlights
+
+- ASAR Integrity fuse now supported on Windows [#40504](https://github.com/electron/electron/pull/40504)
+  - Existing apps with ASAR Integrity enabled may not work on Windows if not configured correctly.
+  - Take a look at our [ASAR Integrity tutorial](https://www.electronjs.org/docs/latest/tutorial/asar-integrity) for more information.
+- Added `WebContentsView` and `BaseWindow`, deprecating & replacing `BrowserView` [#35658](https://github.com/electron/electron/pull/35658)
```

Comment:
We should point to https://www.electronjs.org/docs/latest/tutorial/web-embeds to compare and contrast `WebContentsView` as well.

Reply:
issue(blocking): I don't think the release notes are clear enough about the BrowserView implementation being a JavaScript shim for WebContentsView now.

Reply:
I guess "deprecating and replacing" sounds a bit vague to me as in the old impl doesn't exist in the code anymore.